### PR TITLE
Add some commands

### PR DESCRIPTION
**Pull request by gc**
Adds some commands.

### CODE REVIEW

File: commands/Fun/randquote.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+		for (let i = 0; i < 6; i++) {
+			const fetchedMessages = await msg.channel.messages.fetch({ limit: 100, before: messageBank.last().id });
+			messageBank = messageBank.concat(fetchedMessages);
+		}
+
+		for (let i = 0; i < messageBank.size; i++) {
+			const message = messageBank.random();
+			if (message.author.bot) continue;
+			if (message.content.replace(/\W/g, '').replace(/[0-9]/g, '').length < 20) continue;
```
Comment (kyranet) [inline commands/Fun/randquote.js]:
```javascript
if (message.author.bot || message.content.replace(/[\W0-9]*/g, '').length) continue;
```

Nonetheless, this looks like you want to use `messageBank.find()`.

File: commands/Fun/randquote.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+			const embed = new MessageEmbed()
+				.setDescription(message.content)
+				.setAuthor(message.author.username, message.author.avatarURL());
```
Comment (kyranet) [inline commands/Fun/randquote.js]: displayAvatarURL()

File: commands/Fun/wordcloud.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+		const ctx = FinalImage.getContext('2d');
+		let messageBank = await msg.channel.messages.fetch({ limit: 100 });
+		const wordBank = {};
+		for (let i = 0; i < 20; i++) {
+			const fetchedMessages = await msg.channel.messages.fetch({ limit: 100, before: messageBank.last().id });
+			messageBank = messageBank.concat(fetchedMessages);
+		}
+
+		for (const message of messageBank.values()) {
+			if (!message.content) continue;
```
Comment (kyranet) [inline commands/Fun/wordcloud.js]: This check is useless, the second condition covers this.

File: commands/Tools/followage.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+			.catch(() => { throw `${twitchName} isn't following ${channelName}, or it is banned, or doesn't exist at all.`; });
+
+		const embed = new MessageEmbed()
+			.setColor(6570406)
+			.setAuthor(`${twitchName} has been following ${channelName} for ${days} days.`, logo);
+		return msg.sendEmbed(embed);
+	}
+
+	differenceDays(first, second) {
+		return parseInt((second - first) / (1000 * 60 * 60 * 24));
```
Comment (kyranet) [inline commands/Tools/followage.js]: Useless `parseInt`

**Comment by gc** [comment]
All requested changes have been made

File: commands/Fun/randquote.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+		});
+	}
+
+	async run(msg) {
+		let messageBank = await msg.channel.messages.fetch({ limit: 100 });
+		for (let i = 1; i < messageLimitHundreds; i++) {
+			messageBank = messageBank.concat(await msg.channel.messages.fetch({ limit: 100, before: messageBank.last().id }));
+		}
+
+		for (let i = 0; i < messageBank.size; i++) {
```
Comment (kyranet) [inline commands/Fun/randquote.js]:
```js
const message = messageBank
	.filter(ms => !ms.author.bot && !ms.content.replace(/[\W0-9]*/g, '').length < 20)
	.random();
if (!message) throw 'Could not find a quote';
return msg.sendEmbed(new MessageEmbed()
	.setDescription(message.content)
	.setAuthor(message.author.username, message.author.displayAvatarURL()));
```

Comment (kyranet) [reply commands/Fun/randquote.js]: Implement the same logic (filters) to all other pieces that behave similarly.

Comment (kyranet) [reply commands/Fun/randquote.js]: And the `msg.sendEmbed(new MessageEmbed());` style aswell.

**Comment by gc** [comment]
new requested changes done (and tested)

**Pull request merged**